### ISSUE TITLE
buttons in Framework7 (and possibly other frameworks)

### ISSUE THREAD

**Issue by jwjcmw** [sampotts/plyr#928]
Over in the vue-plyr issues, someone brought up that they were using Framework7, and the buttons were stretching out, which looks to be due to a global button style in the F7 css.  Would it make sense to add:
```css
button {
        font: inherit;
        line-height: inherit;
        width: inherit;  /*this is the added line*/
    }
```
to the plyr base.scss to make sure this isn't an issue?  I can generate a PR if so.
The issue with a screenshot is here: https://github.com/redxtech/vue-plyr/issues/15

**Comment by sampotts** [comment]
I can add `width: auto` as that seems more logical to me as a reset?

**Comment by jwjcmw** [comment]
works for me...thanks!

**Comment by sampotts** [comment]
Will add to the next release (probably later today)

**Comment by sampotts** [comment]
This should be resolved in v3.3.0 👍
